keras/src/trainers/compile_utils.py

```diff
@@ -406,6 +409,8 @@ def from_config(cls, config):
 
 
 class CompileLoss(losses_module.Loss):
+    Loss = namedtuple("Loss", ["path", "loss", "loss_weights", "name"])
```

Comment:
Is this being used? If so, why do we need it?

Reply:
It was meant for clarity for developers. `CompileLoss` builds a list `_flat_losses` of those 4-tuples. I made this namedtuple to enhance understanding of what those tuples represent.
What is your suggestion, shall I use regular tuples?

Reply:
Oh I see. I wasn't aware of the line `CompileLoss.Loss`.
I agree that `namedtuple` is indeed clearer than a regular tuple.